zenith/src/main.rs

```diff
@@ -319,23 +346,43 @@ fn get_branch_infos(env: &local_env::LocalEnv) -> Result<HashMap<ZTimelineId, Br
     Ok(branch_infos)
 }
 
+fn handle_tenant(tenant_match: &ArgMatches, env: &local_env::LocalEnv) -> Result<()> {
+    let pageserver = PageServerNode::from_env(&env);
+    if let Some(tenantid) = tenant_match.value_of("tenantid") {
+        pageserver.tenant_create(&ZTenantId::from_str(tenantid)?)?
```

Comment:
I would expect that if `tenantid` is not provided, pageserver will generate some random id.
Just like in `zenith init --create-tenant`.

Reply:
Also, let's print a message about operation success.

Reply:
I've added tenant autogeneration to cli instead of pageserver. IMO it is more suitable because calling code almost always need to keep track of newly created tenant